ui/DesignSystem/Component/AdditionalActionsDropdown.svelte

```diff
@@ -21,10 +25,18 @@
     item.event();
   };
 
-  export let dataCy = null;
-  export let style = null;
-  export let menuItems = null;
-  export let headerTitle = null;
+  export let menuItems: {
```

Comment:
Could probably be pushed into a ts file and reused wherever we render meus.

Reply:
For now I think this is fine because the type-checker will throw an error if this is used incorrectly by a parent component.